apps/frontend/player/src/pages/PreviewPage/PreviewPage.tsx

```
@@ -1,12 +1,31 @@
-  const isEmbedded = isRunningInIframe();
+  const [searchParams] = useSearchParams();
+  const navigate = useNavigate();
+  const huntId = searchParams.get('huntId');
+
+  // Embedded mode - Editor sends data via SDK
+  if (isInIframe()) {
+    return <EmbeddedPreview />;
+  }
+
+  // Standalone with huntId - redirect to play route
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)